<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="sign_in_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="join_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QPushButton" name="leave_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>280</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Leave</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="username_window">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="user_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>49</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>User</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chat_window">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>631</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="message_input">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>631</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="member_window">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>330</y>
      <width>271</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="group_label">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>190</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Group</string>
    </property>
   </widget>
   <widget class="QLabel" name="chat_label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Chat Content</string>
    </property>
   </widget>
   <widget class="QPushButton" name="send_button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>630</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_out_button">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>140</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Sign Out</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="group_window">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>230</y>
      <width>271</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;Consolas&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="app_label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 48pt &quot;Consolas&quot;;</string>
    </property>
    <property name="text">
     <string>Chat Room</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
